tracer/src/Datadog.Trace/Ci/CiEnvironment/GitCommandGitInfoProvider.cs

```diff
@@ -15,6 +15,8 @@ namespace Datadog.Trace.Ci.CiEnvironment;
 
 internal sealed class GitCommandGitInfoProvider : GitInfoProvider
 {
+    private static readonly char[] GitOutputTrimChars = ['\n', '\r', ' '];
```

Comment:
Interesting, so you explicitly _don't_ want to trim other whitespace like `\t`? Just these three? 🤔